packages/types/surveys.ts

```diff
@@ -206,6 +215,16 @@ const ZSurveyQuestionBase = z.object({
   isDraft: z.boolean().optional(),
 });
 
+export const ZSurveyFileUploadQuestion = ZSurveyQuestionBase.extend({
+  type: z.literal(TSurveyQuestionType.FileUpload),
+  allowMultipleFiles: z.boolean(),
+  maxSizeInMB: z.number().optional(),
+  allowedFileTypes: z.array(ZAllowedFileExtension).optional(),
```

Comment:
we sometimes have `fileExtensions` and sometimes `fileTypes`. Please choose one to be consistent 😊
I think `fileExtensions` is more common